### PR TITLE
Add menu

### PR DESCRIPTION
**Pull request by BagrijRoman**

### CODE REVIEW

File: client/src/components/Menu/Menu.js [hunk 1]
```diff
@@ -1,30 +1,119 @@
...
+
+    this.setState({ currentWidth: innerWidth });
+  };
+
   render() {
     const {
       props: { collapsed },
     } = this;
 
+    const menuItemData = [
```
Comment (ivan-khr85) [inline client/src/components/Menu/Menu.js]: this should be in a separate file

File: client/src/components/Menu/components/MenuItem/MenuItem.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      || menuItemExpanded !== nextMenuItemExpanded
+      || !R.equals(data, nextData);
+  }
+
+  toggleItemExpanded = menuItemExpanded => this.setState({ menuItemExpanded });
+
+  renderMenuItem = ({
+    collapsed, iconKey, label, subItems,
+  }) => {
+    const { menuItemExpanded } = this.state;
```
Comment (ivan-khr85) [inline client/src/components/Menu/components/MenuItem/MenuItem.js]: should be in one destruction

File: client/src/components/Menu/components/MenuItem/MenuItem.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    );
+  };
+
+
+  render() {
+    const {
+      renderMenuItem,
+      props: {
+        collapsed,
+        data: {
```
Comment (ivan-khr85) [inline client/src/components/Menu/components/MenuItem/MenuItem.js]: need to define the default value for data if we will not have this prop

File: client/src/components/Menu/Menu.js (hunk 1 above)
Comment (BagrijRoman) [reply client/src/components/Menu/Menu.js]: done

File: client/src/components/Menu/components/MenuItem/MenuItem.js (hunk 2 above)
Comment (BagrijRoman) [reply client/src/components/Menu/components/MenuItem/MenuItem.js]: done

**Pull request merged**